config/routes.rb

```diff
@@ -11,6 +11,10 @@
      end
   end
 
-  root "api/v1/cars#index"
+  post 'signup', to: 'api/v1/users#create'
+  post '/login', to: 'api/v1/users#login'
+  get '/auto_login', to: 'api/v1/users#auto_login'
```

Comment:
What does this route do? I can understand the login process, but I don't get how we can do a auto_login in front-End part!